app/lib/pre_assembly/digital_object.rb

```diff
@@ -13,28 +14,29 @@ class DigitalObject
              to: :bundle
 
     attr_accessor :container,
-                  :label,
-                  :manifest_row,
-                  :pre_assem_finished,
-                  :source_id
+                  :pre_assem_finished
 
     attr_writer :dor_object
 
     # @param [PreAssembly::Bundle] bundle
     # @param [String] container the identifier (non-namespaced)
     # @param [Array<String>] stageable_items items to stage
     # @param [Array<ObjectFile>] object_files path to files that are part of the object
-    def initialize(bundle, container: nil, stageable_items: nil, object_files: nil)
+    # @param [String] label The label for this object
+    # @param [String] pid The identifier for the item
+    # @param [String] source_id The source identifier
+    # rubocop:disable Metrics/ParameterLists
+    def initialize(bundle, container: nil, stageable_items: nil, object_files: nil,
+                   label: nil, pid: nil, source_id: nil)
```

Comment:
shouldn't the default for label be 'Unknown' based on "setup" method being removed?

Reply:
@ndushay we never initialize it without passing a label (except in the tests).   The `Bundle` always passes a label or an empty string: https://github.com/sul-dlss/pre-assembly/pull/571/files#diff-cda6cd930edb1f3a0ac4fafc58185f2bR86